sdf/A-2084-01.sdf

```diff
@@ -0,0 +1,504 @@
+<sdf version='1.6'>
+  <model name='A-2084-01'>
+    <link name='HEBI/base/INPUT_INTERFACE'>
+      <pose frame=''>0 0 0 0 -0 0</pose>
+      <inertial>
+        <pose frame=''>-0.0142 -0.0031 0.0165 0 -0 0</pose>
+        <mass>0.335</mass>
+        <inertia>
+          <ixx>0.00015</ixx>
+          <ixy>3.41e-05</ixy>
+          <ixz>1.18e-05</ixz>
+          <iyy>0.000255</iyy>
+          <iyz>2.29e-06</iyz>
+          <izz>0.00035</izz>
+        </inertia>
+      </inertial>
+      <collision name='HEBI/base/INPUT_INTERFACE_collision'>
+        <pose frame=''>0 0 0 0 -0 0</pose>
+        <geometry>
+          <mesh>
+            <scale>0.001 0.001 0.001</scale>
+            <uri>model://hebi_description/meshes/X5.stl</uri>
```

Comment:
Did we get these meshes successfully showing up in gazebo when running with these .sdf files?

Reply:
Yes, but I should change them to hebi_common, otherwise we have a dependency on hebi_description non-ROS users may want to avoid. Since the meshes never really change this seems like an ok thing to have duplicated?